docs/try_local_system.md

```diff
@@ -166,6 +170,73 @@ Once the services have been confirmed to be up and running, you can proceed with
    If the command returned a `200 OK` status code, proceed with
    [trying out our APIs](#try-out-our-apis).
 
+## Set up a Nix development environment
+
+A Nix development environment simplifies the setup of required project dependencies. This is available for MacOS, Linux and WSL2 users.
+
+### Install nix
+
+Nix can be installed in numerous ways. If you're new to Nix, use the recommended approach. However, you're free to choose the way you'd like to have nix installed on your device. The aim is to have nix installed with `flakes` support.
+
+#### 1. Using nixos-unified-template [recommended]
+
+*nixos-unified-template* provides an unified Nix configuration template for managing user applications and using `flakes`.
+
+- Head to [nixos-unified-template](https://github.com/juspay/nixos-unified-template?tab=readme-ov-file#on-non-nixos) and follow the instructions.
+
+#### 2. Using nix-installer [minimal]
+
+*nix-installer* provides a quick and reliable way of installing Nix with `flakes`.
+
+- Head to [nix-installer](https://github.com/DeterminateSystems/nix-installer) and follow the instructions.
```

Comment:
We strive to document the process in external repos/pages, so the project README should be as simple/minimal as possible.

```suggestion
### Install nix

We recommend that you install Nix using [the DetSys nix-installer](https://nixos.asia/en/install), which automatically enables flakes.

As an **optional** next step, if you are interested in using Nix to manage your dotfiles and local packages, you can setup [nixos-unified-template](https://github.com/juspay/nixos-unified-template#on-non-nixos).
```